Resources/doc/resource_owners/jira.md

```diff
@@ -7,8 +7,7 @@ Jira instance. Check out the [Atlassian documentation](https://confluence.atlass
 Next configure a resource owner of type `jira` with appropriate
 `client_id`, `client_secret` and `base_url`.
 
-The Client Secret should either be the formatted private key  PEM string
-or a path to the PEM file.
+The Client Secret should either be a path to the private key pem file.
```

Comment:
`s/pem/PEM`

Why this could not be `the formatted private key  PEM string` anymore?